ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverPlugin.java

```diff
@@ -17,8 +17,6 @@
 package com.netflix.ndbench.plugin.cass;
 
 import com.datastax.driver.core.*;
```

Comment:
These are the best comments I have found for Git commits: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html